src/main/java/rosenfeld/scrabble/Scrabble.java

```diff
@@ -38,6 +38,9 @@ public String getDefinition(String word) {
         return definition == null ? "" : definition;
     }
 
+    public Map<String, String> getWordsToDefinitions() {
```

Comment:
You shouldnt be using this method and instead using the other method to look up the word that you created.

Reply:
> 

Ok I created a DictionaryService to read my text file. I found a way to do it differently because its not a JSON file so it reads the file and puts the words/definitions into a map differently. I use the get method on the dictionary HashMap created in Servlet but am still getting a null definition.